purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/components/TimelineComponent.kt

```diff
@@ -99,6 +103,14 @@ class PartialTimelineComponent(
     val padding: Padding? = null,
     @get:JvmSynthetic
     val margin: Padding? = null,
+) : PartialComponent
+
+@InternalRevenueCatAPI
+@Poko
+@Serializable
+class PartialTimelineComponentItem(
+    @get:JvmSynthetic
+    val visible: Boolean? = null,
     @get:JvmSynthetic
-    val items: List<Item>? = null,
-)
+    val connector: TimelineComponent.Connector? = null,
```

Comment:
Note that I'm not adding overrides to the `title`, `description` or `icon` here. I believe those should be overrides to the respective `TextComponent` and `IconComponent`